pages/docs/adapters/calculix/adapter-calculix-get-calculix.md

```diff
@@ -126,15 +126,15 @@ After building, make sure that you make yaml-cpp discoverable by setting e.g. yo
 
 ### Get the source
 
-Once the libraries are installed, you can finally install Calculix with preCICE adapter (adapt the `VERSION` in the link, see the beginning of the [adapter's README.md](https://github.com/precice/calculix-adapter/blob/master/README.md) to find out which one you need).
+Once the libraries are installed, you can finally install Calculix with preCICE adapter. Note that the adapter version needs to be the same as the CalculiX version (replace `{{site.calculix_version}}` below).
 
```

Comment:
Maybe mention that we support older versions, if necessary ?

Reply:
I think this is a bigger discussion and I would like to defer it to the point we have a release strategy.